docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -326,6 +326,14 @@ Additionally it is possible to set:
   `<Response_Header_1, ..., Response_Header_n>` to specify headers to pass to backend once authentication request completes.
 * `nginx.ingress.kubernetes.io/auth-request-redirect`:
   `<Request_Redirect_URL>`  to specify the X-Auth-Request-Redirect header value.
+* `nginx.ingress.kubernetes.io/auth-snippet`:
```

Comment:
Make sure it is added to the main table in this section.
Add a note that its optional and that auth-url is required.